compiler/lib/eval.ml

```diff
@@ -86,9 +82,21 @@ let eval_prim x =
   | Eq, [ Int i; Int j ] -> bool Targetint.(i = j)
   | Neq, [ Int i; Int j ] -> bool Targetint.(i <> j)
   | Ult, [ Int i; Int j ] -> bool (Targetint.(j < zero) || Targetint.(i < j))
-  | Extern name, l -> (
-      let name = Primitive.resolve name in
+  | Extern name', l -> (
+      let name = Primitive.resolve name' in
       match name, l with
```

Comment:
Since these operations make sense in Wasm as well, maybe you should do:
```ocaml
match if String.equal name "%identity-ints-repr" then name' else name with
```